generated/api/api.module.ts

```diff
@@ -8,8 +8,8 @@ import {
 import { HttpClient } from '@angular/common/http';
 import { ApiConfiguration, ApiConfigurationParams } from './api-configuration';
 
-import { AlgorithmService } from './services/algorithm.service';
 import { AlgorithmRelationTypeService } from './services/algorithm-relation-type.service';
+import { AlgorithmService } from './services/algorithm.service';
```

Comment:
hmm, did you use the current ng-openapi-fork?
All of that should be sorted by now.

Reply:
I did rerun it after npm install, maybe it looks good now.